lib/screens/weekplan_screen.dart

```diff
@@ -34,15 +36,22 @@ class WeekplanScreen extends StatelessWidget {
     weekplanBloc.setWeek(_week, _user);
   }
 
-  /// The WeekplanBloc that contains the currently chosen week
+  /// The WeekplanBloc that contains the current chosen weekplan
   final WeekplanBloc weekplanBloc = di.getDependency<WeekplanBloc>();
   final UsernameModel _user;
   final WeekModel _week;
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: GirafAppBar(title: 'Ugeplan'),
+      appBar: GirafAppBar(
+        title: 'Ugeplan',
+        appBarIcons: <AppBarIcon, VoidCallback>{
+          AppBarIcon.edit: () => weekplanBloc.toggleEditMode(),
+          AppBarIcon.settings: null,
```

Comment:
The empty callback should be safer

```suggestion
          AppBarIcon.settings: () {},
```